src/Symfony/Component/Console/Application.php

```diff
@@ -1026,7 +1026,7 @@ protected function getDefaultInputDefinition()
 
             new InputOption('--help', '-h', InputOption::VALUE_NONE, 'Display this help message'),
             new InputOption('--quiet', '-q', InputOption::VALUE_NONE, 'Do not output any message'),
-            new InputOption('--verbose', '-v|vv|vvv', InputOption::VALUE_NONE, 'Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug'),
+            new InputOption('--verbose', '-v|vv|vvv', InputOption::VALUE_OPTIONAL, 'Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug', 1),
```

Comment:
that's a BC break, as `bin/console foo -v bar` will now consider `bar` as the value of the verbose option and not as an argument (which is precisely why that change was reverted in the past)

Reply:
what about deprecating the `--verbose=1` syntax then ?

because it still allow to make commands tests succeed with a bad option configuration.
should the `verbose` long option be handled as an exception when validating here ?

something like : 
- reverting back the configuration of verbose option
- validatin in ArrayInput with : 
```
-      if (!\in_array($value, [null, true], true) && !$option->acceptValue()) {
+      if (
+           !\in_array($value, [null, true], true)
+           && (!$option->acceptValue() && 'verbose' !== $name)
+      ) {
           throw new InvalidOptionException(sprintf('The "--%s" option does not accept a value.', $name));
       }
```

Reply:
Thanks @stof for pointing this BC break in my initial solution.

I forcepushed with a new approach: 
- still validating the values for options in mode VALUE_NONE
- but handling the `verbose` long option as an exception to accept values.

Only thing i dont like here is now the `verbose` option of the application is known in low level objects ArrayInput & ArgvInputs.